.github/workflows/build-image.yml

```diff
@@ -28,7 +28,7 @@ jobs:
           echo "REPOBASE=$REPOBASE" >> $GITHUB_ENV
           echo "IMAGETAG=$IMAGETAG" >> $GITHUB_ENV
           echo "IMAGENAME=$IMAGENAME" >> $GITHUB_ENV
-          podman build -t ${IMAGENAME}:${IMAGETAG} .
+          podman build -env GITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }} -t ${IMAGENAME}:${IMAGETAG} .
```

Comment:
```suggestion
          podman build --env GITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }} -t ${IMAGENAME}:${IMAGETAG} .
```

Reply:
I'd suggest to change the name of the secret so we can set it globally inside the organization.
Something like `FONTAWSOME_GITHUB_TOKEN`